drivers/serial/uart_stm32.c

```diff
@@ -395,6 +395,54 @@ static void uart_stm32_poll_out(struct device *dev,
 	LL_USART_TransmitData8(UartInstance, (u8_t)c);
 }
 
+static int uart_stm32_err_check(struct device *dev)
+{
+	/**
+	 * LPUART and U(S)ART have the same base and use the same
```

Comment:
Cf comment in 13163

Reply:
Removed LPUART comment completely, as PR #13163 adds a more generic comment.